README.md

```diff
@@ -13,17 +13,83 @@ A Cornell University CIT Custom Development starter kit package for Laravel.
+   >**NOTE**: If you have GitHub CLI installed, you can immediately add this to GitHub as a repo with the following commands (be sure to replace the "your-app-name" references with your project info):
+   > ```shell
+   > cd your-app-name
+   > git add . && git commit -m "Initial commit"
+   > git branch -m main
+   > gh repo create --private CU-CommunityApps/CD-your-app-name
+   > git push --set-upstream origin main
+   > ```
+   
 2. Composer require the LaravelStarterKit
    ```shell
-   composer require cornell-custom-dev/laravel-starter-kit
+   composer require --update-no-dev cornell-custom-dev/laravel-starter-kit
    ```
+   Similar to the `create-project` option, the `--update-no-dev` keeps us from adding baggage to the vendor dir.
+
 3. Install the Starter Kit
    ```shell
    php artisan starterkit:install
    ```
-   This will publish a `README.md` and `.lando.yml` file to the base directory, configured on the project settings. It will also publish HTML/CSS/JS assets from [cwd_framework_lite](https://github.com/CU-CommunityApps/cwd_framework_lite) and a set of [view components](https://laravel.com/docs/9.x/blade#layouts-using-components) that can be used to begin a layout (see `views/cwd-framework-index.blade.php` for example usage).
+   This will publish `README.md`, `.env.example`, `.gitignore`, and `.lando.yml` files to the base directory, configured on the project settings and update the `composer.json` file to match. It will also publish HTML/CSS/JS assets from [cwd_framework_lite](https://github.com/CU-CommunityApps/cwd_framework_lite) and a set of [view components](https://laravel.com/docs/9.x/blade#layouts-using-components) that can be used to begin a layout (see `views/cwd-framework-index.blade.php` for example usage).
+   > **NOTE**: The install step updates `.gitignore` so that the vendor directory is no longer excluded. The next commit will be large because it includes everything in the vendor directory.
+   >```shell
+   > git add . && git commit -m "Starter Kit install"
+   > git push
+   >```
+
+4. Testing the site<br>
+   You can confirm the site is working with Lando, since the Starter Kit install process adds a `.lando.yml` file.
+   ```shell
+   lando start
+   ```
+   Then visit https://your-app-name.lndo.site and you should see the default Laravel page. To see the Laravel Starter Kit example page, edit `/resources/views/welcome.blade.php` to be:
+    ```blade
+    @include('cwd-framework-index.blade.php')
+    ```
+
+## Deploying a site
+Once a Media3 site has been created, you have confirmed you can reach the default site via a web browser, and you have access to the site login by command line, the code can be deployed.
+
+You will likely need to map the `php` command to the correct version by editing `~/.bashrc` to include this alias (for this to take effect, run `source ~/.bashrc` or just log in again):
+```shell
+# User specific aliases and functions
+alias php="/opt/cpanel/ea-php81/root/usr/bin/php"
```

Comment:
I've been using alias php="/usr/local/bin/ea-php81" on all of my sites

Reply:
Thanks, updated!